lib/diaspora_federation/validators/rules/numeric.rb

```diff
@@ -0,0 +1,21 @@
+module Validation
+  module Rule
+    class Numeric
```

Comment:
Please use the numeric rule from the valid-gem for this: https://github.com/zombor/Validator/blob/master/lib/validation/rule/numeric.rb